### PR TITLE
fix(stemmer): fix stemmer installation path

### PR DESCRIPTION
**Pull request by micheleriva**
hotfix for #126

### CODE REVIEW

**Comment by micheleriva** [comment]
@codyzu any chance you can fix this? 😔

File: package.json [hunk 1]
```diff
@@ -19,10 +19,10 @@
     "format": "prettier --write \"**/*.{ts,js}\"",
     "lint": "pnpm run format && eslint . --ext .js,.ts",
     "prepare": "husky install && npm run build",
-    "build:cjs": "tsc --project tsconfig.cjs.json",
-    "build:module": "tsc --project tsconfig.esm.json",
+    "build:cjs": "tsc --project tsconfig.cjs.json && cp -R stemmer/cjs/en.js dist/stemmer/cjs/en.js && cp -R stemmer/cjs/en.d.ts dist/stemmer/cjs/en.d.ts",
+    "build:module": "tsc --project tsconfig.esm.json && cp -R stemmer/esm/en.js dist/stemmer/esm/en.js && cp -R stemmer/esm/en.d.ts dist/stemmer/esm/en.d.ts",
     "build:browser": "tsc --project tsconfig.browser.json",
-    "build": "rimraf ./dist && concurrently 'npm:build:module' 'npm:build:cjs' 'npm:build:browser'",
+    "build": "rimraf ./dist && mkdir -p dist/stemmer/{cjs,esm} && concurrently 'npm:build:module' 'npm:build:cjs' 'npm:build:browser'",
```
Comment (mateonunez) [inline package.json:25]: Replace that main build command with:

```js
"build": "rimraf ./dist && concurrently 'npm:build:module' 'npm:build:cjs' 'npm:build:browser' && cp -R stemmer/ dist/stemmer",
```

The `dist` path will contain _browser_, _cjs_, _esm_ and _stemmer_.

Is not necessary change `build:cjs` and build:esm` commands.

Comment (codyzu) [reply package.json:25]: this probably would have fixed the issue, the downside is that `cp` is not windows compatible. Probably better to use something like https://www.npmjs.com/package/copyfiles

In any case, I think a better solution is to let the typescript compiler handle this. See #128

**Comment by codyzu** [comment]
> @codyzu any chance you can fix this? 😔

@micheleriva I took a different approach in #128. With the new approach, you can _only_ include the esm build from snowball 🎉 

I recommend closing this in favor of #128 when you have a chance to look

**Comment by micheleriva** [comment]
Thanks Cody you're a hero

**Pull request closed without merging**